README.md

```diff
@@ -79,28 +87,32 @@ const trie = new Trie(db, stateRoot)
 
 const address = 'AN_ETHEREUM_ACCOUNT_ADDRESS'
 
-const data = await trie.get(address)
-const acc = new Account(data)
-
-console.log('-------State-------')
-console.log(`nonce: ${new BN(acc.nonce)}`)
-console.log(`balance in wei: ${new BN(acc.balance)}`)
-console.log(`storageRoot: ${bufferToHex(acc.stateRoot)}`)
-console.log(`codeHash: ${bufferToHex(acc.codeHash)}`)
-
-let storageTrie = trie.copy()
-storageTrie.root = acc.stateRoot
-
-console.log('------Storage------')
-const stream = storageTrie.createReadStream()
-stream
-  .on('data', (data) => {
-    console.log(`key: ${bufferToHex(data.key)}`)
-    console.log(`Value: ${bufferToHex(rlp.decode(data.value))}`)
-  })
-  .on('end', () => {
-    console.log('Finished reading storage.')
-  })
+async function test() {
+  const data = await trie.get(address)
+  const acc = new Account(data)
+
+  console.log('-------State-------')
+  console.log(`nonce: ${new BN(acc.nonce)}`)
+  console.log(`balance in wei: ${new BN(acc.balance)}`)
+  console.log(`storageRoot: ${bufferToHex(acc.stateRoot)}`)
+  console.log(`codeHash: ${bufferToHex(acc.codeHash)}`)
+
+  let storageTrie = trie.copy()
+  storageTrie.root = acc.stateRoot
+
+  console.log('------Storage------')
+  const stream = storageTrie.createReadStream()
+  stream
+    .on('data', (data) => {
+      console.log(`key: ${bufferToHex(data.key)}`)
+      console.log(`Value: ${bufferToHex(rlp.decode(data.value))}`)
+    })
+    .on('end', () => {
+      console.log('Finished reading storage.')
+    })
+}
+
+await test()
```

Comment:
Removing the await since it won't execute unless it's in an async function itself.
```suggestion
test()
```